frontend/awx/resources/templates/JobTemplateInputs.tsx

```diff
@@ -95,9 +94,9 @@ function JobTemplateInputs(props: { jobtemplate?: JobTemplateForm }) {
         isRequired={!isInventoryPrompted}
       />
       <PageFormProjectSelect<JobTemplateForm>
-        name="summary_fields.project.name"
+        name="summary_fields.project"
```

Comment:
I'm seeing an issue in the usage of the PageFormProjectSelect component in the `frontend/awx/access/roles/AddRolesForm.tsx` form.

This particular error `cannot read properties of undefined (reading summary_fields)` is not seen with the old select component. 

![image](https://github.com/ansible/ansible-ui/assets/43621546/c59171cc-e5ee-472f-a2ec-16ceada650f5)

Reply:
I'm seeing a similar error with the JT and WFJT select components `Cannot read properties of undefined (reading 'summary_fields')`. On the `main` branch too there is a related error (`Cannot read properties of undefined (reading 'id')`) for the old JT select component.
![image](https://github.com/ansible/ansible-ui/assets/43621546/8cd92381-9997-4bef-b21f-2b98f1290b63)

![image](https://github.com/ansible/ansible-ui/assets/43621546/addfbf49-fd22-4adf-b9b6-9c648495bb01)